canonical-websites/maas.io
# Update install page styling

## Don't merge

 - Waiting on #278 to land first

## Done

- Updated page to Vanilla 1.8.0
- Turns out copy hasn't changed on this page :D

## QA

- Pull code, run `./run`, navigate to http://0.0.0.0:8006/install
- Check everything looks good

## Issue / Card

Fixes #266
